location/migrations/0002_static_data.py

```diff
@@ -0,0 +1,33 @@
+from django.db import migrations
+
+
+def create_initial_locations(apps, schema_editor):
+    Location = apps.get_model('location', 'Location')
+    Location.objects.create(name='Bloomfield', city='Tel-Aviv-Yaffo', street='Hatkuma', street_number=10,
+                            indoor=False, description='A soccer field in tel-aviv')
+    Location.objects.create(name='Country Club TA A', city='Tel-Aviv-Yaffo', street='Rabbeinu Yerucham',
+                            street_number=2, indoor=False, description='At the western area')
+    Location.objects.create(name='Country Club TA B', city='Tel-Aviv-Yaffo', street='Rabbeinu Yerucham',
+                            street_number=2, indoor=True, description='At the eastern area')
+    Location.objects.create(name='Sportsplex Arena', city='Haifa', street='Ben Guriun', street_number=5,
+                            indoor=False, description='Near the local high-school')
+    Location.objects.create(name='The Ultimate Sports Hub', city='Haifa', street="Ha'Atzma'ut", street_number=33,
+                            indoor=True, description='For all inside activities')
+
+    Location.objects.create(name='The All-In-One Sports Dome', city='Jerusalem', street='Yitzhak Rabin',
+                            street_number=42, indoor=False, description='For all outside activities')
+
+    Location.objects.create(name='Holmes Place Family A', city='Ashdod', street='Barak Ben Avinoam', street_number=10,
+                            indoor=True, description='At the northern area')
+    Location.objects.create(name='Holmes Place Family B', city='Ashdod', street='Barak Ben Avinoam', street_number=10,
+                            indoor=True, description='At the southern area')
```

Comment:
instead of repeating the Location.objects.create a few times maybe you can do something like:
```suggestion
            location_test_data = [
            ('Bloomfield', 'Tel-Aviv-Yaffo', 'Hatkuma', 10,  False, 'A soccer field in tel-aviv'),
            ('Country Club TA A', 'Tel-Aviv-Yaffo', 'Rabbeinu Yerucham', 2, False, 'At the western area'),
            ...
        ]

        with transaction.atomic():
            for name, city, street, street_number, indoor, description in location_test_data:
                location = Location(name=name, city=city, street=street, street_number=street_number,
                            indoor=indoor, description=description)                
                location.save()
```